widget/embedded/src/components/SwapDetails/SwapDetails.tsx

```diff
@@ -0,0 +1,495 @@
+        onCancel:
+          swap.status === 'running' ? () => setModalState('cancel') : undefined,
+        suffix: swap.status !== 'running' && (
+          <Button
+            variant="ghost"
+            type="error"
+            size="small"
+            onClick={() => setModalState('delete')}>
+            <Typography size="medium" variant="label" color="error">
+              {i18n.t('Delete')}
+            </Typography>
+          </Button>
+        ),
+      }}
+      noPadding>
+      <Container>
+        <HeaderDetails>
+          <div className="row">
+            <Typography variant="label" size="large" color="neutral900">
+              {`${i18n.t('Request ID')}:`}
+            </Typography>
+            <div className="request-id">
+              <Typography variant="label" size="small" color="neutral900">
+                {requestId}
+              </Typography>
+              <IconButton
+                variant="ghost"
+                onClick={handleCopy.bind(null, requestId || '')}>
+                <CopyIcon size={16} color="gray" />
+              </IconButton>
+            </div>
+          </div>
+          <div className="row">
+            <Typography variant="label" size="large" color="neutral900">
+              {`${i18n.t('Created at')}:`}
+            </Typography>
+            <Typography variant="label" size="small" color="neutral900">
+              {swapDate}
+            </Typography>
+          </div>
+        </HeaderDetails>
+        <div className="output">
+          <RouteCost
+            fee={numberToString(
+              String(totalFee),
+              AMOUNT_DECIMALS,
+              AMOUNT_DECIMALS
+            )}
+            time={minutes}
+            steps={numberOfSteps}
+          />
+          <TokenAmount
+            direction="horizontal"
+            type="input"
+            price={{
+              value: numberToString(
+                swap.inputAmount,
+                AMOUNT_DECIMALS,
+                AMOUNT_DECIMALS
+              ),
+              usdValue: numberToString(
+                String(
+                  parseFloat(swap.inputAmount) *
+                    (getUsdPrice(
+                      firstStep.fromBlockchain,
+                      firstStep.fromSymbol,
+                      firstStep.fromSymbolAddress,
+                      tokens
+                    ) || 0)
+                ),
+                MIN_DECIMALS,
+                AMOUNT_DECIMALS
+              ),
+            }}
+            token={{
+              displayName: steps[0].from.token.displayName,
+              image: steps[0].from.token.image,
+            }}
+            chain={{ image: steps[0].from.chain.image }}
+          />
+          <Separator />
+          <TokenAmount
+            direction="horizontal"
+            type="output"
+            price={{
+              value: numberToString(
+                !!lastStep.outputAmount
```

Comment:
I think you could rewrite it like this:

`lastStep.outputAmount ?? lastStep.expectedOutputAmountHumanReadable`

Reply:
fixed.

Reply:
fixed.